wazuhapp/generate_wazuh_app.sh

```diff
@@ -32,7 +33,7 @@ build_package(){
     # Build the Docker image
     docker build -t ${CONTAINER_NAME} ./Docker/
     # Build the Wazuh Kibana app package using the build docker image
-    docker run --rm -t  -v ${TMP_DIR}:/source -v "${DESTINATION}":/wazuh_app ${CONTAINER_NAME} ${wazuh_version} ${kibana_version} ${app_revision} 
+    docker run --rm -t  -v ${TMP_DIR}:/source -v "${DESTINATION}":/wazuh_app ${CONTAINER_NAME} ${wazuh_version} ${kibana_version} ${app_revision} ${checksum}
```

Comment:
Same as I said previously in the splunkapp script. Use uppercase in the variables.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/9d09627160a93091dae44ea05cd9ca3429cd8f21